inventory.yml.sample

```diff
@@ -94,6 +94,8 @@ all:
       - "{{ repo_root_path }}/ssh_public_key.pub"
       - ~/.ssh/id_rsa.pub
 
+    ssh_key_dest_base_dir: /home/redhat
```

Comment:
Please can you add a short comment, similar to the one below (or extend that), to clarify under what circumstances a user might wish to modify this value, and to what.

Reply:
Ok, I would have thought that was self-explanatory tbh.